### PR TITLE
ignore NCM:: namespace for all unittests

### PR DESCRIPTION
**Pull request by stdweird**
Requires new buildtools with https://github.com/quattor/maven-tools/pull/77 in them

### CODE REVIEW

**Comment by stdweird** [comment]
This requires a new release of the buildtools

**Comment by ned21** [comment]
Code looks fine but can you explain why this is necessary in the commit log please?

**Comment by stdweird** [comment]
rpmlint errors left
```
ncm-ncd.noarch: W: non-standard-group Quattor
ncm-ncd.noarch: E: non-standard-dir-perm /var/log/ncm 750
ncm-ncd.noarch: W: no-manual-page-for-binary quattor-list
ncm-ncd.noarch: W: no-manual-page-for-binary quattor-configure
ncm-ncd.noarch: W: no-manual-page-for-binary quattor-unconfigure
1 packages and 0 specfiles checked; 1 errors, 4 warnings.
```

@ned21 @jrha @jouvin why do we have the non-standard permissions on the log dir?


wrt pom.xml cleanup
* /var/tmp/ncm gave rpmlint error, and looked unused, so removed (see #62 )
* /var/lock/quattor and /var/run/quattor-components also gave rpmlint error, but this is handled by the code already (as they are volatile directories in EL7 and solaris)

**Comment by stdweird** [comment]
@ned21 i rephrased the first commit

**Comment by jrha** [comment]
@stdweird no idea about the non-standard log dir permissions, seems like a mistake to me.

**Comment by jrha** [comment]
`/var/log/ncm` was added by @piojo in 28383aeb161a01e93e1a13d14dc3fd0b187afd81, no mention of any particular reason for log permissions in that commit.

File: pom.xml [hunk 1]
```diff
@@ -1,403 +1,345 @@
...
-	      </mapping>
-	    </mappings>
-	  </configuration>
-	</plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>rpm-maven-plugin</artifactId>
+          <configuration>
+            <summary>NCD is the front-end for invoking NCM components</summary>
+            <url>https://github.com/quattor/CCM/tree/master</url>
```
Comment (ned21) [inline pom.xml]: Wrong URL? Seems to reference the CCM tree, not ncm-ncd

File: pom.xml [hunk 2]
```diff
@@ -1,403 +1,345 @@
...
+            <name>ncm-ncd</name>
+            <mappings combine.self="override">
+              <mapping>
+                <directory>/var/log/ncm</directory>
+                <directoryIncluded>true</directoryIncluded>
+                <filemode>750</filemode>
```
Comment (ned21) [inline pom.xml]: I would prefer 755. This is what we have at the moment.

**Comment by stdweird** [comment]
@ned21 fixed, permissions set to 755, now rpmlint gives
```
ncm-ncd.noarch: W: non-standard-group Quattor
ncm-ncd.noarch: W: no-manual-page-for-binary quattor-list
ncm-ncd.noarch: W: no-manual-page-for-binary quattor-configure
ncm-ncd.noarch: W: no-manual-page-for-binary quattor-unconfigure
1 packages and 0 specfiles checked; 0 errors, 4 warnings.
```

**Pull request merged**